spec/index.bs

```diff
@@ -1894,6 +1900,111 @@ dictionary GPUVertexAttributeDescriptor {
 };
 </script>
 
+## {{GPUPipelineBase/getBindGroupLayout()|getBindGroupLayout}} ## {#get-bind-group-layout}
+
+<dl dfn-type="method" dfn-for="GPUPipelineBase">
+    : <dfn>getBindGroupLayout(unsigned long index)</dfn>
+    ::
+        <div algorithm="GPUProgrammablePipeline.getBindGroupLayout(unsigned long index)">
+            1. If |index| is greater or equal to {{GPULimits/maxBindGroups}} :
+
+                1. Throw a {{RangeError}}.
+
+            1. If |this|.getBindGroupLayout has already been called with |index|:
+
+                1. Return the same object.
+
+            1. If |this| is an error:
+
+                1. Return an error {{GPUBindGroupLayout}}.
+
+            1. Let |bindings| be an empty sequence.
+            1. For each {{GPUProgrammableStageDescriptor}} |stage| in the descriptor for |this|:
+
+                1. For each resource |resource| in |stage|.{{GPUProgrammableStageDescriptor/module}}
+                    for the |stage|.{{GPUProgrammableStageDescriptor/entryPoint}} entry-point:
```

Comment:
I'm having trouble understanding what this has to do with `entryPoint`

Reply:
The list of bindings used by a pipeline depends on the entryPoint, for example in SPIRV `OpEntryPoint` lists all the interface variables used by the entry point.